OpenOversight/app/main/views.py

```diff
@@ -565,21 +576,23 @@ def add_department():
 @login_required
 @admin_required
 def edit_department(department_id):
-    jsloads = ["js/jquery-ui.min.js", "js/deptRanks.js"]
```

Comment:
We never edit this, so I deleted the variable and added it to the return response.